controllers/snapshot/snapshot_adapter.go

```diff
@@ -127,41 +127,16 @@ func (a *Adapter) EnsureRerunPipelineRunsExist() (controller.OperationResult, er
 	}
 	testStatuses.ResetStatus(scenarioName)
 
-	if shouldScenarioRunInEphemeralEnv(integrationTestScenario) {
-		allEnvironments, err := a.loader.GetAllEnvironments(a.client, a.context, a.application)
-		if err != nil {
-			a.logger.Error(err, "Failed to get all environments.")
-			return controller.RequeueWithError(err)
-		}
-
-		_, err = a.ensureEphemeralEnvironmentForScenarioExists(integrationTestScenario, testStatuses, allEnvironments, ScenarioOptions{IsReRun: true})
-		if err != nil {
-			if h.IsEnvironmentNotInNamespaceError(err) {
-				a.logger.Error(err, "environment doesn't exist in the same namespace as integrationScenario at all, try again after creating environment",
-					"integrationTestScenario.Name", integrationTestScenario.Name)
-				// remove label to prevent indefinite failures, users must restart test manually again
-				if err = gitops.RemoveIntegrationTestRerunLabel(a.client, a.context, a.snapshot); err != nil {
-					return controller.RequeueWithError(err)
-				}
-				return controller.StopProcessing()
-			}
-			return controller.RequeueWithError(fmt.Errorf("failed to ensure existence of the environment for scenario %s: %w", integrationTestScenario.Name, err))
-		}
-
-	} else {
```

Comment:
Why are we removing else statement content there? I'm missing how is it connected to ephemeral env here.